sdcm/cluster.py

```diff
@@ -2591,7 +2591,7 @@ def _gen_nodetool_cmd(self, sub_cmd, args, options):
     # pylint: disable=inconsistent-return-statements
     def run_nodetool(self, sub_cmd, args="", options="", timeout=None,
                      ignore_status=False, verbose=True, coredump_on_timeout=False,
-                     warning_event_on_exception=None, error_message="", publish_event=True):
+                     warning_event_on_exception=None, error_message="", publish_event=True, retry=1):
```

Comment:
zero mean don't retry, right ?
why should it be the default ?

I can't think of any nodetool we'll want to retry automatically

Reply:
And `1` is a default value for it in `remoter.run()`